TypeScript: v1.FirestoreAdminClient is not constructable.

Caused by the new typings added in https://github.com/googleapis/nodejs-firestore/commit/47238a926471dee8bdeaa38bcb5f772c7f20349f.

It seems that `v1.FirestoreAdminClient` (as well as `v1.FirestoreClient` and `v1beta1.FirestoreClient`) is exported as instance instead of class.

#### Environment details

  - OS: macos
  - Node.js version: 12.19.0
  - npm version: 6.14.8 (using yarn v2.4.1)
  - `@google-cloud/firestore` version: 

#### Steps to reproduce

In TypeScript, try to instantiate `firestore.v1.FirestoreAdminClient`.

`tsc` output:
```
src/functions/backup-firestore/handler.ts:15:22 - error TS2351: This expression is not constructable.
  Type 'FirestoreAdminClient' has no construct signatures.

15   const client = new firestore.v1.FirestoreAdminClient()
                        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


Found 1 error.
```

--- a/types/firestore.d.ts
+++ b/types/firestore.d.ts
@@ -1967,7 +1967,7 @@ declare namespace FirebaseFirestore {
    * @deprecated Use v1 instead.
    */
   export const v1beta1: {
-    FirestoreClient: import('./v1beta1/firestore_client').FirestoreClient;
+    FirestoreClient: typeof import('./v1beta1/firestore_client').FirestoreClient;
   };
 
   /**
@@ -1975,8 +1975,8 @@ declare namespace FirebaseFirestore {
    * API and the underlying Firestore v1 RPCs.
    */
   export const v1: {
-    FirestoreClient: import('./v1/firestore_client').FirestoreClient;
-    FirestoreAdminClient: import('./v1/firestore_admin_client').FirestoreAdminClient;
+    FirestoreClient: typeof import('./v1/firestore_client').FirestoreClient;
+    FirestoreAdminClient: typeof import('./v1/firestore_admin_client').FirestoreAdminClient;
   };
 
   /**

